sarc/cli/acquire/storages.py

```diff
@@ -18,22 +20,29 @@ class AcquireStorages:
     cluster_names: list[str] = field(alias=["-c"], default_factory=list)
     dry: bool = False
 
+    tracer = get_tracer("AcquireStorages")
+
     def execute(self) -> int:
         cfg = config()
 
         for cluster_name in self.cluster_names:
-            print(f"Acquiring {cluster_name} storages report...")
+            with self.tracer.start_as_current_span("cluster") as span:
```

Comment:
Ça serait une bonne opportunité de protéger cette partie du code pour qu'une exception ne fasse pas planter la boucle au complet. Si on remplace le `with self.tracer` par un contextmanager qui catch les exceptions ça va permettre aussi de setter automatiquement les exceptions dans la trace.

Reply:
done in https://github.com/mila-iqia/SARC/pull/107/commits/c58821f7d85b4c1027caadb8f671cab0ef3b787d